lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -647,8 +678,10 @@ export abstract class ClientApplication {
     getAccountByUsername(userName: string): AccountInfo|null {
         const allAccounts = this.getAllAccounts();
         if (!StringUtils.isEmpty(userName) && allAccounts && allAccounts.length) {
+            this.logger.verbosePii(`Returning signed-in accounts matching username: ${userName}`);
```

Comment:
For every Pii log, can you also add a non-pii message next to it?

Reply:
Done